arm/Microsoft.DesktopVirtualization/applicationGroupsResources/applications/.parameters/parameters.json

```diff
@@ -28,11 +28,8 @@
                 }
```

Comment:
General comment: We should probably create an issue to make this module deploy only a single instance, right?

Reply:
In general yes. However probably not worty as it should be part of the restructure and be tested together with the appgroup.

Reply:
You are completly right. My bad.